homeassistant/components/switchbee/climate.py

```diff
@@ -106,6 +107,13 @@ def __init__(
         self._attr_temperature_unit = HVAC_UNIT_SB_TO_HASS[device.temperature_unit]
         self._attr_hvac_modes = [HVAC_MODE_SB_TO_HASS[mode] for mode in device.modes]
         self._attr_hvac_modes.append(HVACMode.OFF)
+        self._attr_supported_features = (
+            ClimateEntityFeature.TARGET_TEMPERATURE | ClimateEntityFeature.FAN_MODE
+        )
```

Comment:
This part seems duplicated now (compared to line 90-92)

Reply:
Thanks. Forgot to clean it of from class 👍